### PR TITLE
add less/netcat/patch to containers and the tests

### PR DESCRIPTION
**Pull request by erain**

### CODE REVIEW

File: skylib/package_names.bzl [hunk 1]
```diff
@@ -229,6 +235,13 @@ def _jessie_java_no_ca_certs_package_names():
 def _jessie_java_package_names():
     return _jessie_java_no_ca_certs_package_names() + ["ca-certificates-java"]
 
+def _jessie_less_package_names():
+    return [
+        "libc6",
+        "libtinfo5",
+        "debianutils",
```
Comment (nlopezgi) [inline skylib/package_names.bzl]: this should probably include the "less" package?

File: skylib/package_names.bzl [hunk 2]
```diff
@@ -247,6 +260,11 @@ def _jessie_lib_cpp_package_names():
         "linux-libc-dev",
     ]
 
+def _jessie_netcat_package_names():
+    return [
+        "netcat-traditional",
```
Comment (nlopezgi) [inline skylib/package_names.bzl]: add "netcat" package here as well

File: skylib/package_names.bzl [hunk 3]
```diff
@@ -259,6 +277,11 @@ def _jessie_openssh_client_package_names():
         "openssh-client",
     ]
 
+def _jessie_patch_package_names():
+    return [
+        "libc6",
+    ]
```
Comment (nlopezgi) [inline skylib/package_names.bzl]: add "patch"

File: skylib/package_names.bzl (hunk 1 above)
Comment (erain) [reply skylib/package_names.bzl]: Thanks! I am still doing testing (that's why I haven't requested review), and I just found the issue you indicated!

**Pull request merged**